sdk/src/wallet/update.rs

```diff
@@ -25,12 +26,34 @@ where
     crate::wallet::Error: From<S::Error>,
     crate::client::Error: From<S::Error>,
 {
-    /// Set the alias for the wallet.
-    pub async fn set_alias(&self, alias: &str) -> crate::wallet::Result<()> {
-        let mut wallet_data = self.data_mut().await;
-        wallet_data.alias = Some(alias.to_string());
+    /// Update the wallet address with a possible new Bech32 HRP and clear the inaccessible incoming transactions.
+    pub(crate) async fn update_wallet_address_hrp(&self) -> crate::wallet::Result<()> {
```

Comment:
I wouldn't have `wallet` in a `Wallet` method name

Reply:
okay